### PR TITLE
updates

### PR DESCRIPTION
**Pull request by geramirez**
- Split deploy-reports into deploy-reports and deploy-report-schedules
- Use powershell syntax for deploy-sites

### CODE REVIEW

File: deploy-site.ps1 [hunk 1]
```diff
@@ -21,24 +21,41 @@ Remove-Item .\_site\package.json
...
-Pop-Location
-exit $winscpResult
+$session = New-Object WinSCP.Session
+try
+{
+    # Connect
+    $session.Open($sessionOptions)
+    Write-Host "Uploading files"
+    $synchronizationResult = $session.SynchronizeDirectories(
+      [WinSCP.SynchronizationMode]::Remote, "$CurrentDirectory\analytics-site\_site\", "analytics", $true)
```
Comment (cailyncodes) [inline deploy-site.ps1]: Can the third parameter be relative?

File: deploy-reports.ps1 [hunk 2]
```diff
@@ -49,7 +47,4 @@ try
 finally
 {
     $session.Dispose()
-}
-    
-
-ssh $UserName@$HostIP "powershell .\$ProjectLocation\create_tasks.ps1 -ProjectLocation $ProjectLocation -OutputDirectory $OutputDirectory"
```
Comment (cailyncodes) [inline deploy-reports.ps1]: Is the idea then to run `deploy-report-schedules.ps1` separately? Or should this line be replace with an invocation to `deploy-report-schedules.ps1`? I think either is fine, just wanted to know what your plan was.

File: deploy-site.ps1 (hunk 1 above)
Comment (geramirez) [reply deploy-site.ps1]: For deploy site, we probably want it to always be true.

File: deploy-reports.ps1 (hunk 2 above)
Comment (geramirez) [reply deploy-reports.ps1]: I realized that we will have to update reports without updating their schedules so I split them.

File: deploy-site.ps1 (hunk 1 above)
Comment (cailyncodes) [reply deploy-site.ps1]: Sorry I meant the "analytics" parameters (versus "\analytics")

File: deploy-reports.ps1 (hunk 2 above)
Comment (cailyncodes) [reply deploy-reports.ps1]: Ah yeah that makes sense. Good call!

**Pull request merged**